server.ts

```diff
@@ -479,6 +481,15 @@ function saveToCache(req, page: any) {
   }
 }
 
+/**
+ * Check if status code is different from 2XX
+ * @param statusCode
+ */
+function hasNotSucceeded(statusCode) {
+  const rgx = new RegExp(/20+/);
```

Comment:
Shouldn't we make sure this *starts* with `20`?

In other words, I wonder if this should be more strict.. like `/^20+/`.  Otherwise it's possible that "420" would match.  It's an "unofficial" code but it used to exists in Spring, see https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#Unofficial_codes

Reply:
I verified locally that this code still works if you change this line to 
```
const rgx = new RegExp(/^20+/);
```

While I think your code is *likely* OK, I'd recommend we make this Regex more strict to avoid any unexpected issues later on.

Reply:
done